eth-bytecode-db/eth-bytecode-db/src/verification/db.rs

```diff
@@ -10,6 +10,14 @@ use sea_orm::{
 };
 use std::collections::BTreeMap;
 
+#[cfg(feature = "test-bins")]
+pub async fn _insert_test_data(
```

Comment:
do you need this?

Reply:
I'm not sure that we need it in this PR. But I still added it to make it explicit which function we may use to insert data to the database when testing